process/block/metablock.go

```diff
@@ -320,12 +321,6 @@ func (mp *metaProcessor) processEpochStartMetaBlock(
 		return err
 	}
 
-	defer func() {
```

Comment:
Why did we remove this? In case we hit any of the following errors we do not revert the account state making impossible to (re)sync the next block

Reply:
it is called the defer from ProcessBlock, It would have been called twice.

Reply:
oh